structs.go

```diff
@@ -784,6 +784,20 @@ func (m *Member) Mention() string {
 	return "<@!" + m.User.ID + ">"
 }
 
+// A MemberEdit holds Member Field data for a member edit.
+type MemberEdit struct {
+	Nick  string   `json:"nick,omitempty"`
+	Roles []string `json:"roles,omitempty"`
```

Comment:
You may need `*[]string` here. Refer to https://play.golang.org/p/m-coc5VIfPv.

Reply:
:disappointed: Right, of course - no way to remove all roles from someone w/ `omitempty`, argh!